File: ruby/FireMissiles.rb
puts "DISCLAIMER: This will not actually fire missiles"
print "Who do you want missiles sent to? "
location = gets.chomp
puts "You want missiles sent to #{location}? Y/N"
confirm = gets.chomp

if confirm == "Y" or confirm == "y"
puts "Contacting computer... SUCCESS"
puts "Getting location... SUCCESS"
puts "Missiles launching in 3..2..1"
puts "Missiles launched"
puts "Missiles successfully sent to #{location}."
elsif confirm == "N" or confirm == "n"
puts "Operation aborted."
else
puts "Invalid response."
end

File: python/python2/FireMissiles.py
person = raw_input("Who should the missiles be sent to? ")
confirm = raw_input("You want missiles sent to " + person + "? Y/N: ")

if confirm == "Y" or confirm == "y":
	print "Contacting computer... SUCCESS"
	print "Getting location... SUCCESS"
	print "Missiles launching in 3..2..1"
	print "Missiles launched"
	print "Missiles sucessfully sent to " + person + "."
elif confirm == "N" or confirm == "n":
	print "Operation aborted."
else:
	print "Invalid response."
